src/test/java/org/takes/rs/RsWithCookieTest.java

```diff
@@ -69,20 +87,26 @@ public void addsCookieToResponse() throws IOException {
      */
     @Test
     public void addsMultipleCookies() throws IOException {
+        final String setcookie = "Set-Cookie: xoo=value?;Path=/;";
+        final String xoo = "xoo";
+        final String bar = "bar";
+        final String value = "value?";
         MatcherAssert.assertThat(
             new RsPrint(
                 new RsWithCookie(
                     new RsWithCookie(
                         new RsEmpty(),
-                        "foo", "works?", "Path=/"
+                        xoo,
+                        value,
+                        RsWithCookieTest.PATH
```

Comment:
@dalifreire I think we could have another unique path here and get rid of the shared `PATH` variable